infra/cifuzz/cifuzz.py

```diff
@@ -524,6 +524,13 @@ def is_project_sanitizer(sanitizer, oss_fuzz_project_name):
 
   # Simple parse to prevent adding pyYAML dependency.
   with open(project_yaml, 'r') as file_handle:
-    if sanitizer + '\n' in file_handle.read():
-      return True
+    file_data = file_handle.read()
+
+  # TODO(metzman): Replace this with proper handling of project.yaml files.
+  if 'sanitizers:\n' not in file_data:
+    logging.info('No sanitizers defined in project.yaml, assuming %s is valid.',
+                 sanitizer)
+    return True
+  if sanitizer + '\n' in file_data:
```

Comment:
To cover something like
```
- memory:
    experimental: True
```
it should probably look for `sanitizer:?\n` via a regex maybe.

Reply:
I'm going to switch this to something non-hacky that uses pyyaml.
But what do you think should be the behavior for this? I think we should allow `experimental` sanitizers in CIFuzz.

Reply:
Oh I get the point your making. This fails now. Lemme try to fix with your suggestion.

Reply:
I don't want to do regex because the sanitizer can be the last in the file.
Since this is a hack to make hacky code deal with an edge case, I'm fine if the validation isn't 100% perfect.

Reply:
Agreed. Looking for `sanitizer` seems to be better in this particular case.